fhir-provider/pom.xml

```diff
@@ -37,9 +37,14 @@
             <artifactId>jakarta.ws.rs-api</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-rt-frontend-jaxrs</artifactId>
+            <scope>provided</scope>
```

Comment:
sorry, i must've missed this one on my first pass through.  can this one get test-scoped too or we need this one?

Reply:
I'll try it out.

Reply:
I updated it to test and the build still works, so looks good.